interfaces/apparmor/spec.go

```diff
@@ -48,6 +49,46 @@ func (spec *Specification) AddSnippet(snippet string) {
 	}
 }
 
+func snippetFromLayout(layout *snap.Layout) string {
+	mountPoint := layout.Snap.ExpandSnapVariables(layout.Path)
```

Comment:
We should probably at least check that this starts with '/'.

Reply:
We check below (we check if it is absolute and clean)

Reply:
Update: this is now done in the snap validation function (merged separately)